generator/Makefile

```diff
@@ -73,19 +89,45 @@ $(MIBDIR)/ARISTA-SW-IP-FORWARDING-MIB:
 	@echo ">> Downloading ARISTA-SW-IP-FORWARDING-MIB"
 	@curl -s -o $(MIBDIR)/ARISTA-SW-IP-FORWARDING-MIB -L "$(ARISTA_URL)/ARISTA-SW-IP-FORWARDING-MIB.txt"
 
-$(MIBDIR)/.cisco_v2:
+$(MIBDIR)/cisco_v2:
 	@echo ">> Downloading cisco_v2"
+	@mkdir $(MIBDIR)/cisco_v2
 	@curl -s -L $(CISCO_URL) \
-		| tar -C $(MIBDIR) --strip-components=3 -zxv
-	@touch $(MIBDIR)/.cisco_v2
-
-$(MIBDIR)/KEEPALIVED-MIB.txt:
-	@echo ">> Downloading keepalived"
-	@curl -s -o $(MIBDIR)/KEEPALIVED-MIB.txt -L $(KEEPALIVED_URL)
-
-$(MIBDIR)/mikrotik.mib:
-	@echo ">> Downloading mikrotik"
-	@curl -s -o $(MIBDIR)/mikrotik.mib -L $(MIKROTIK_URL)
+		| tar -C $(MIBDIR)/cisco_v2 --strip-components=3 -zxv
+	cp mibs/cisco_v2/AIRESPACE-REF-MIB.my mibs/AIRESPACE-REF-MIB
```

Comment:
A for loop would be a bit clearer here I think

Reply:
Maybe, I wanted to just keep it simple.